bin/mypy

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+python -m mypy -p warehouse
```

Comment:
Let's go ahead and add this to `bin/lint` instead.

Reply:
Reverted inclusion of this file and added to `bin/lint` in recent commit.